apps/server/src/infra/tsp-client/tsp-client-factory.ts

```diff
@@ -1,5 +1,6 @@
+import { DomainErrorHandler } from '@core/error';
 import { AxiosErrorLoggable, ErrorLoggable } from '@core/error/loggable';
-import { Logger } from '@core/logger';
+import { TspAccessTokenLoggableError } from '@infra/sync/strategy/tsp/loggable/tsp-access-token.loggable-error';
```

Comment:
The tsp client is used in sync. We should not import from sync in the client. 
Maybe the loggable should be placed in client module.

Reply:
Done